Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/weaponryworks.yml

```diff
@@ -35,6 +39,10 @@
     WeaponRifleNovaliteC1: 4294967295
     WeaponShotgunDoubleBarreled: 4294967295
     WeaponRevolverArgenti: 4294967295
+    WeaponPistolPollock: 4294967295
+    WeaponRifleVulcan: 4294967295
+    WeaponRevolverFitz: 4294967295
+    WeaponRevolverFaith: 4294967295
```

Comment:
This items shouldn't be available from POI vendomat at round start, please remove them